README.md

```diff
@@ -46,12 +46,20 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
+* Install Docker Community Edition and docker-compose
```

Comment:
I originally thought the alternation of ordered and unordered lists was getting busy looking, but I think you're right that it's clearer if the outermost list markup reflects that these steps need to happen in order.